apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.scss

```
@@ -368,7 +368,7 @@ cdk-tree {
                     padding-top: 10px;
                     padding-bottom: 10px;
 
-                    min-width: 140px;
+                    min-width: 154px;
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)